### PR TITLE
Handling video outstream in smartadserver adapter.

### PR DESCRIPTION
**Pull request by tadam75**
## Type of change
- [x] Feature

## Description of change
Adding video outstream support in SmartAdServer adapter.

### CODE REVIEW

**Comment by mlapeyre3** [comment]
Hello @mike-chowla, did you have time to review our PR?
Thanks!

**Comment by mlapeyre3** [comment]
Hllo @gglas / @mike-chowla any news? Can we expect a review of our PR next week? I need to inform our partners about it, just need some visibility :)
Thanks!

File: modules/smartadserverBidAdapter.md [hunk 1]
```diff
@@ -94,4 +94,41 @@ Please reach out to your Technical account manager for more information.
...
+					playerSize: [640, 480]
+				}
+			},
+			renderer: {
+				url: 'https://acdn.adnxs.com/video/outstream/ANOutstreamVideo.js',
+				render: function (bid) {
+					ANOutstreamVideo.renderAd({
+						targetId: bid.adUnitCode,
+						adResponse: bid
+					});
```
Comment (jsnellbaker) [inline modules/smartadserverBidAdapter.md:116]: I think an update is needed here.  In the documentation [here](https://docs.prebid.org/dev-docs/show-outstream-video-ads.html#renderers), it has the actual renderer call wrapped around a queue handler to ensure the called gets executed at the right time.

When I was testing with these exact params, I was getting an error during the rendering step that the `ANOustreamVideo` did not exist.  I think it just means this call here was happening too early before the script fully deployed.  Can you make an update here (just so the example is right)?

**Review by jsnellbaker — CHANGES_REQUESTED** [review]
Hi @tadam75 

When I was first testing the changes with the example params, I was getting a bid back.  However after a few tries, the bid stopped coming back.  Is there some type of capping put in place?  If so, is there some way I can clear it?

Please confirm when you have the chance.

File: modules/smartadserverBidAdapter.md (hunk 1 above)
Comment (tadam75) [reply modules/smartadserverBidAdapter.md]: Thanks for the doc, I fixed it.

**Comment by tadam75** [comment]
> Hi @tadam75
> 
> When I was first testing the changes with the example params, I was getting a bid back. However after a few tries, the bid stopped coming back. Is there some type of capping put in place? If so, is there some way I can clear it?
> 
> Please confirm when you have the chance.

Indeed we have a kind of throttling server side, that we can't really override 😔
It didn't occur to me during my tests but it can definitely happen.

**Review by jsnellbaker — APPROVED** [review]
Thanks for the clarification, I got it to work.  LGTM

**Pull request merged**